README.md

```diff
@@ -46,12 +46,12 @@ The hash function takes in the following parameters:
 - **version** : Argon2 algorithm version number.
 - **encoding** : Encoding for the returned hash type ('raw', 'hex' or 'b64').
 
-For assistance with parameter selection refer to the [draft RFC](https://www.ietf.org/id/draft-irtf-cfrg-argon2-10.txt), in particular "Chapter 4: Parameter Choice".
+For assistance with parameter selection refer to the [draft RFC](https://www.rfc-editor.org/rfc/rfc9106.html#section-4), in particular "Chapter 4: Parameter Choice".
```

Comment:
Notice that it's an anchor link to section 4, should the paragraph reflect this?

Reply:
Either is fine, thank you for updating the documentation 👍